.eslintrc.js

```diff
@@ -54,22 +40,26 @@ module.exports = {
         '.mocharc.js',
         '.eslintrc.js',
         '.prettierrc.js',
-        '.template-lintrc.js',
```

Comment:
This file does not exist. All template lint files are in packages directories.

Reply:
Without this, `yarn workspace <packagename> lint:js` fails

Reply:
@Gaurav0 do you mind expanding on this?

I just tested the command you mentioned with both of the workspaces that have a `lint:js` defined and it seemed to be fine:

These are the commands I ran (on this branch):

```bash
yarn workspace adapter-encapsulation-test-app lint:js
yarn workspace debug-encapsulation-test-app lint:js
```

Reply:
You are correct, it is working on your branch. You may have fixed something, or I may have gotten the error another way. Feel free to ignore me for now.